package.json

```diff
@@ -147,7 +147,7 @@
     "mobxpromise": "github:cbioportal/mobxpromise#v1.0.2",
     "node-sass": "^4.9.3",
     "numeral": "^2.0.6",
-    "oncoprintjs": "^1.1.9",
+    "oncoprintjs": "github:cbioportal/oncoprintjs#61c9f7196262082635ed16f51abc5798bc313edb",
```

Comment:
@alisman can we do this?

Reply:
I do believe this works based in a little googlery.  Try it! If it works ... it works!!!

Reply:
it does seem to work... i was just wondering if you had had issues w it with jitpack or something bc i remember there was some reason you preferred to use tags

Reply:
most of the examples seem to use tags. probably more kosher to use tags but not important to me.

Reply:
I would only use tags if you can guarantee they are reliable. E.g. 1.1.10 should be backwards compatible with 1.1.9 since it is a minor release. If this is not the goal, then what you propose above should be fine.